rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -202,6 +204,8 @@ class PublisherBase
   
   std::vector<std::shared_ptr<QOSEventHandlerBase>> event_handlers_;
 
+  std::vector<std::shared_ptr<QOSEventHandlerBase>> event_handlers_;
```

Comment:
Duplicate definition of `event_handlers_`?

Reply:
Good catch, thanks, the merge was a little hairy